src/os/bluestore/KernelDevice.cc

```diff
@@ -645,9 +645,11 @@ int KernelDevice::aio_write(
       ++injecting_crash;
     } else {
       bl.prepare_iov(&aio.iov);
-      for (unsigned i=0; i<aio.iov.size(); ++i) {
-	dout(30) << "aio " << i << " " << aio.iov[i].iov_base
-		 << " " << aio.iov[i].iov_len << dendl;
+      if (cct->_conf->subsys.should_gather(ceph_subsys_bdev, 30)) {
```

Comment:
nit: IMO a bit better to define
const int LEVEL = 30;
and use it for both if and dout..